kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerConnectionSuite.scala

```diff
@@ -80,7 +80,9 @@ class KyuubiOperationPerConnectionSuite extends WithKyuubiServer with HiveJDBCTe
       assert(executeStmtResp.getOperationHandle === null)
       val errMsg = executeStmtResp.getStatus.getErrorMessage
       assert(errMsg.contains("Caused by: java.net.SocketException: Connection reset") ||
-        errMsg.contains(s"Socket for ${SessionHandle(handle)} is closed"))
+        errMsg.contains(s"Socket for ${SessionHandle(handle)} is closed") ||
+        errMsg.contains("Socket is closed by peer") ||
+        errMsg.contains("SparkContext was shut down"))
```

Comment:
Could the `errMsg` contain `Broken pipe (Write failed)`?

Reply:
I think no, because the error is likely occurs on reading